### ISSUE TITLE
Aria-hidden attributes lacking values

### ISSUE THREAD

**Issue by Cerchie** [Virtual-Coffee/virtualcoffee.io#252]
## Describe the bug

Aria labels on social media icons on home page are `aria-hidden=""`. I imagine what we want is `aria-hidden="true"`. Feel free to assign to me or to someone who'd like to get started in on OS contributions. 💟 

## To Reproduce

1. Install lighthouse https://developers.google.com/web/tools/lighthouse
2. Click on lighthouse to run report

## Expected behavior

Should be getting a 93 on a11y due to aria-labels on social media links.

## Screenshots

<img width="1033" alt="Screen Shot 2021-05-31 at 12 53 57 PM" src="https://user-images.githubusercontent.com/54046179/120235916-f1aee100-c20f-11eb-8e7b-da17ff53490e.png">

## Desktop (please complete the following information):

- OS: Catalina
 - Browser Chrome

**Comment by danieltott** [comment]
Thanks @Cerchie it's all you!

**Comment by danieltott** [comment]
Closed via #257 - thank you @Cerchie!!

![](https://media4.giphy.com/media/3oEdvb8F5b16u0okpy/giphy.gif)
